bug_change_status_page.php

```diff
@@ -158,7 +159,7 @@
 
 <?php
 	$t_current_resolution = $t_bug->resolution;
-	$t_bug_is_open = $t_current_resolution < $t_resolved;
+	$t_bug_is_open = $t_current_resolution < $t_resolution_fixed;
```

Comment:
> I think that documentation of $g_bug_resolution_fixed_threshold is wrong.
> ```
> Resolutions above this threshold
> ```
>  
> I think it should be
>  ```
> Resolutions above and including this threshold
>  ```
> Do you agree?

Yes.

> I don't see how we could have a clean solution without 
> - reordering resolutions (which would mean data migration) or
> - being able to configure individual resolutions as fixed/not fixed instead of the two threshold options.
>  
> I think this is beyond the scope of this PR, which is just to prevent strange resolution changes when closing issues, caused by comparing status values with resolution values.

Agreed. I vaguely remember discussing this before... Do you know if there is already an issue to track this (I didn't check) ?